### PR TITLE
AEROGEAR-2121

### PR DESCRIPTION
**Pull request by StephenCoady**
### Motivation

JIRA: https://issues.jboss.org/browse/AEROGEAR-2121

### Changes

Changed the device trust controllers and views to use the new security check results from the sdk as opposed to the old service.

### Verification
See the steps in the linked [PR](https://github.com/aerogear/aerogear-ios-sdk/pull/109)

### CODE REVIEW

File: ios-showcase-template/services/DevicetrustService.swift [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (aidenkeating) [inline ios-showcase-template/services/DevicetrustService.swift]: We should fix this

Comment (StephenCoady) [reply ios-showcase-template/services/DevicetrustService.swift]: wow missed that file altogether

**Comment by wtrocki** [comment]
@StephenCoady Did you disabled build for this PR or it just did not trigger?
Can you rebase?

**Comment by StephenCoady** [comment]
@wtrocki I didn't disable the build and it is rebased already

**Comment by aidenkeating** [comment]
@StephenCoady This looks good to me, happy to verify whenever you're ready

**Comment by StephenCoady** [comment]
@aidenkeating I removed the modals on the trust page and I made all changes to the sdk. Would you mind giving verification a stab on this if you have the chance?

File: ios-showcase-template/deviceTrust/DeviceTrustViewController.swift [hunk 2]
```diff
@@ -108,18 +95,16 @@ class DeviceTrustViewController: UITableViewController {
...
-
         // set the text colouring
-        if detection.detected {
+        if(detection.passed) {
+            cell.textLabel?.text = detection.name + " not detected"
+            cell.textLabel?.textColor = GREEN_COLOR
+            cell.imageView?.image = UIImage(named: "ic_detected_false")
 
+        } else {
+            cell.textLabel?.text = detection.name + " detected"
```
Comment (aidenkeating) [inline ios-showcase-template/deviceTrust/DeviceTrustViewController.swift]: This seems to be wrong for device lock check. Currently when the check fails it says "Device lock detected" and when it passes it says "Device lock not detected".

**Review by rachael-oregan — APPROVED** [review]
Verified the checks work

**Comment by StephenCoady** [comment]
@wtrocki any idea why the build isn't running? It might be something to do with my fork/permissions on this repo? I created a dummy PR here: https://github.com/aerogear/ios-showcase-template/pull/21 and I got the same result. Could you create a similar one and see if it builds?

**Comment by jhellar** [comment]
@StephenCoady @wtrocki when I was creating this repo (for the merged app), I haven't configured circle-ci for it. I think that it does not trigger builds on PRs, but a build is triggered when a PR is merged. I don't have permissions to check circle-ci settings. @wtrocki can you please check?

**Comment by wtrocki** [comment]
Done. If you push new commit build should finish. Circleci is configured. I did not had checked option to build forked branches.

**Comment by StephenCoady** [comment]
thanks @wtrocki

**Comment by wtrocki** [comment]
@StephenCoady I have verified your changes on the device and and 2 different emulators. I did not had chance to simulate jailbreak.
I rebased your changes so you could start your morning with pleasant click on merge button. 
Hope you do not mind.

screen shot 2018-05-22 at 6 32 41 pm

**Review by wtrocki — APPROVED** [review]

**Pull request merged**